File: even_sum_count.py
# Print even numbers from 50 to 100
# Print 5 numbers per line
# Show total count and sum

count = 0
sum = 0

for i in range(50, 101, 2):
    print(i, end=" ")
    count += 1
    sum += i
    if count % 5 == 0:
        print()

print("\nCount:", count)
print("Sum:", sum)

File: even_sum_count.cpp
// Print even numbers from 50 to 100
// Print 5 numbers per line
// Show total count and sum

#include <iostream>
using namespace std;

int main() {
    int count = 0, sum = 0;

    for (int i = 50; i <= 100; i += 2) {
        cout << i << " ";
        count++;
        sum += i;
        if (count % 5 == 0) cout << endl;
    }

    cout << "\nCount: " << count << endl;
    cout << "Sum: " << sum << endl;

    return 0;
}
